config/knex/migrations/20180625211133_create_room_table.js

```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('room', function(table) {
+        table.increments('id').primary()
+        table.integer('building_id').unsigned().references('id').inTable('building')
+        .onUpdate('CASCADE')
+        .onDelete('CASCADE')
+        table.string('room_floor').notNullable()

```

Comment:
What do you think about this `room_floor' field? 

The setup's a bit weird given that for HAAS there's room numbers G56 and 257, and all the LWSN rooms in question are in the basement, i.e., B156. 

The `room_floor' field might be redundant since we already handle `room_numbers' as strings. 

Maybe skip that one or not force it to have a value with notNullable?